Makefile

```diff
@@ -86,9 +86,9 @@ build/ansible-operator:
 ##@ Dev image build
 
 # Convenience wrapper for building all remotely hosted images.
-.PHONY: image-build
 IMAGE_TARGET_LIST = ansible-operator
-image-build: build $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
+image-build: BUILD_GOOS=linux
```

Comment:
Up to you, but you may also consider including `BUILD_GOOS` in the `BUILD_DIR` path. There may be some go toolchain optimizations macos users gain on binary rebuilds when switching back and forth between native host builds and linux image builds.

Reply:
That's a great suggestion. Also, updated the PR to promote commandline vars that is much approach better than the previous commit.